source/UIAUtils.py

```diff
@@ -203,3 +205,11 @@ def getValue(self,ID,ignoreMixedValues=False):
 			raise UIAMixedAttributeError
 		return val
 
+def shouldUseUIAConsole():
+	"Determines whether to use UIA in the Windows Console."
+	setting = config.conf['UIA']['winConsoleImplementation']
+	if setting == "legacy":
+		return False
+	elif setting == "UIA":
+		return True
+	return isWin10(1809) and not isWin10(1903, atLeast=False)

```

Comment:
I think I prefer this to be False for windows 10 1903 and above, until we can make sure that only 1903 will be affected by this.

Reply:
@michaelDCurran How confident are you that this will be fixed in 2003? We're at least 8.5 months from release...